### PR TITLE
[ADD][8.0] video preview functionality

### PR DESCRIPTION
**Pull request by techspawn**

### CODE REVIEW

File: website_upload_video/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# © 2017 Techspawn Solutions (http://techspawn.com)
```
Comment (pedrobaeza) [inline website_upload_video/__init__.py]: Don't add your copyright to non-modified files (I also don't put it on `__init__.py` files - it's a nonsense)

File: website_upload_video/controllers/main.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# © 2017 Techspawn Solutions (http://techspawn.com)
```
Comment (pedrobaeza) [inline website_upload_video/controllers/main.py]: Don't add this copyright

File: website_upload_video/controllers/__init__.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# © 2017 Techspawn Solutions (http://techspawn.com)
```
Comment (pedrobaeza) [inline website_upload_video/controllers/__init__.py]: Don't add this copyright

File: website_upload_video/__openerp__.py [hunk 4]
```diff
@@ -8,6 +9,7 @@
     'category': 'Website',
     'version': '8.0.1.0.0',
     'author': 'ONESTEiN BV,Odoo Community Association (OCA)',
+              'Techspawn Solutions'
```
Comment (pedrobaeza) [inline website_upload_video/__openerp__.py]: With your little change, you deserve to be on the contributors section of the README, but not in the co-author list I think.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@tarteo, you were the original author. Can you please review?

**Comment by techspawn** [comment]
@pedrobaeza Please review the code. We have made the changes that you have requested.

File: website_upload_video/static/src/js/website_upload_video.js [hunk 5]
```diff
@@ -8,6 +8,12 @@
     
     website.editor.VideoDialog.include({
         events: _.extend({}, website.editor.VideoDialog.prototype.events, {
+            // Display Uploaded Video
+            'click .play_video-class':function(e){
+
+     document.getElementById("display_video").innerHTML = "<video width='1000' controls style='margin-top:25px;'>" +'<source src="http://localhost:8069/web/binary/saveas?model=ir.attachment&field=datas&id='+this.get('attachment_id')+'" type="video/mp4">' + "</video>" ;
```
Comment (tarteo) [inline website_upload_video/static/src/js/website_upload_video.js:14]: - localhost:8069 will not work for clients except the server itself.
- Please make it easier to read

File: website_upload_video/static/src/xml/website_upload_video.xml [hunk 6]
```diff
@@ -8,7 +8,15 @@
                 <a class="btn btn-primary filepicker pull-left o_website_upload_video_btn" href="#">
                     Upload a video from your computer
                 </a>
+                
+            <!-- Play video button visible when the video is uploaded -->
+                  <a class=" btn btn-primary play_video-class hidden o_website_upload_success" style='margin-left:5px'>Play Video</a>
```
Comment (tarteo) [inline website_upload_video/static/src/xml/website_upload_video.xml:13]: Could you make the button like the other buttons? 'Preview' and not primary (btn-primary)

This is what I meant with 'the other buttons': 
image

**Review by tarteo — CHANGES_REQUESTED** [review]
Thanks! Can you also please check the tab indent?

**Comment by simahawk** [comment]
Closing as there is no answer for more than one year. 
Feel free to reopen in needed.
Thanks!

**Pull request closed without merging**